### PR TITLE
Clean up maps and add new S1 and S2 pattern maps

### PR DESCRIPTION
**Pull request by l-althueser**
This PR will remove and add some files in the data directory.

1. New S1 and S2 patterns maps with MC v2.1.0 + geo update including PMT window hit R and angle dependence as shown in this [notebook](https://github.com/XENON1T/MCAnalysisScripts/blob/master/PerPMT_LCE_Time/Xe1T_MC_perPMT.ipynb). I opened an issue (#653) for further questions about the S1 patterns. The S2 patterns map is generated exactly as in previous versions (@weiyuehuan pls validate with the [notebook](https://github.com/XENON1T/MCAnalysisScripts/blob/master/PerPMT_LCE_Time/Xe1T_MC_perPMT.ipynb)).
2. The fitted patterns map for S2 patterns have to be updated. @pelssers can you do this or is there any script that I could add to this repo [on Github](https://github.com/XENON1T/MCAnalysisScripts/blob/master/PerPMT_LCE_Time)?
3. Many of the maps were already outdated and not used in PAX. These outdated versions were still used in FAX and the config files prepared in #652. I tried to update all of them to the most recent versions.
4. I think that we have to change the names of the maps also in HAX before we can merge this. (see https://github.com/XENON1T/hax/pull/186)

Data files removed:

- XENON1T_s2_xy_patterns_top_sr0.json.gz (newer version in PAX)
- XENON1T_s2_xy_patterns_top_updatedgeo.json.gz (newer version in PAX)
- XENON1T_s2_xy_patterns_top_v0.3.0.json.gz (newer version in PAX)
- XENON1T_s2_xy_patterns_top_v2.2.0.json.gz (newer version in PAX)
- s2_xy_map_v2.1.json (newer version in PAX)
- transverse_field_rz_correction_12kv.json.gz (not needed since PAX 6.2.0)

I searched and replaced all occurrences of the map name in the PAX repository. This should work out of the box but is not tested (since I don't know how ..).

I could make a separate PR if you don't want to rename the maps.

### CODE REVIEW

File: examples/lce_map_tools/convert_lce.py [hunk 1]
```diff
@@ -131,4 +131,4 @@ def convert_lce_map(filename, outname, n_channels=98, tpc_radius=15.5):
 
 
 
-convert_lce_map("xy-lce-map2.5mm.dat", "s2_xy_patterns_map_XENON100_Xerawdp0.4.5.json.gz")
+convert_lce_map("xy-lce-map2.5mm.dat", "XENON100_s2_xy_patterns_Xerawdp0.4.5.json.gz")
```
Comment (tunnell) [inline examples/lce_map_tools/convert_lce.py]: __[Flake8]__

too many blank lines (3)

File: examples/lce_map_tools/convert_lce.py [hunk 2]
```diff
@@ -129,6 +129,4 @@ def convert_lce_map(filename, outname, n_channels=98, tpc_radius=15.5):
         bla = json.dumps(output)
         outfile.write(bla.encode())
 
-
-
-convert_lce_map("xy-lce-map2.5mm.dat", "s2_xy_patterns_map_XENON100_Xerawdp0.4.5.json.gz")
+convert_lce_map("xy-lce-map2.5mm.dat", "XENON100_s2_xy_patterns_Xerawdp0.4.5.json.gz")
```
Comment (tunnell) [inline examples/lce_map_tools/convert_lce.py]: __[Flake8]__

expected 2 blank lines after class or function definition, found 1

Comment (tunnell) [inline examples/lce_map_tools/convert_lce.py]: __[Flake8]__

no newline at end of file

File: examples/lce_map_tools/convert_lce.py [hunk 3]
```diff
@@ -129,6 +129,5 @@ def convert_lce_map(filename, outname, n_channels=98, tpc_radius=15.5):
         bla = json.dumps(output)
         outfile.write(bla.encode())
 
-
-
-convert_lce_map("xy-lce-map2.5mm.dat", "s2_xy_patterns_map_XENON100_Xerawdp0.4.5.json.gz")
+
```
Comment (tunnell) [inline examples/lce_map_tools/convert_lce.py]: __[Flake8]__

indentation contains mixed spaces and tabs

Comment (tunnell) [inline examples/lce_map_tools/convert_lce.py]: __[Flake8]__

indentation contains tabs

Comment (tunnell) [inline examples/lce_map_tools/convert_lce.py]: __[Flake8]__

blank line contains whitespace

**Comment by tunnell** [comment]
Conflict?

**Comment by feigaodm** [comment]
Hi @l-althueser , can you check this again and resolve the conflicts? Not sure if some more maps has been added. Thanks!

**Comment by l-althueser** [comment]
I will try to solve the conflicts and revisit all files again. Give me some hours ;)

**Comment by l-althueser** [comment]
The PR in PAX and HAX (https://github.com/XENON1T/hax/pull/186) are updated and should be checked by someone else. It is not easy to keep track on all map changes but I visited all commits since the last update of the PR and included everything that was missing.

**Comment by feigaodm** [comment]
Can each map producer help to check whether individial maps are properly renamed and used?
S1 LCE @berget2 
S2 LCE @adambrown1 
S1 AFT @darrylmasson 
Pos FDC @jingqiangye 
FAX @jhowl01 
Single Electron @katrinamiller

**Comment by katrinamiller** [comment]
@feigaodm & @l-althueser - the new naming convention for the single electron maps looks OK!

**Comment by jhowl01** [comment]
Fine for me as well, thanks for checking

**Comment by jingqiangye** [comment]
3D FDC maps are correct.

**Comment by berget2** [comment]
S1 LCE looks good

**Comment by l-althueser** [comment]
Updated to match https://github.com/XENON1T/hax/pull/186

**Pull request merged**